.github/workflows/pyrealm_ci.yaml

```diff
@@ -6,8 +6,8 @@ jobs:
   qa:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-python@v4
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
```

Comment:
The actions are used in more than one job, so we need to update `test` and `docs_build` as well as `qa`.

Reply:
> The actions are used in more than one job, so we need to update `test` and `docs_build` as well as `qa`.

Thanks for the catch! I've updated these ones now.